src/CommunityToolkit.Maui.UnitTests/Mocks/MockDrawingViewHandler.cs

```diff
@@ -3,15 +3,16 @@
 using CommunityToolkit.Maui.Core.Extensions;
 using CommunityToolkit.Maui.Core.Handlers;
 using CommunityToolkit.Maui.Core.Views;
+using CommunityToolkit.Maui.Views;
 using Microsoft.Maui.Handlers;
 
 namespace CommunityToolkit.Maui.UnitTests.Mocks;
 
-public class MockDrawingViewHandler : ViewHandler<IDrawingView, object>, IDrawingViewHandler
+public sealed class MockDrawingViewHandler(IPropertyMapper mapper) : ViewHandler<IDrawingView, DrawingView>(mapper), IDrawingViewHandler
 {
 	IDrawingLineAdapter adapter = new DrawingLineAdapter();
 
-	public static readonly PropertyMapper<IDrawingView, MockDrawingViewHandler> DrawingViewPropertyMapper = new(ViewMapper)
+	public static readonly PropertyMapper<IDrawingView, MockDrawingViewHandler> DrawingViewPropertyMapper = new()
```

Comment:
(Assuming I did finally fix it) Removing `PropertyMapper` here was the fix. 

.NET MAUI was crashing when attempting the mapping for  `[nameof(IView.Clip)]` = MapClip`. 

Since we're not testing anything in `IView`, we don't need to pass `ViewHandler.ViewMapper` into our property mapper on `MockDrawingViewHandler`.

Reply:
Good catch, I guess we might have to revisit this if we do ever test the anything relating to `IView` but for now this looks good.